src/main/java/net/sf/jabref/model/database/DuplicationChecker.java

```diff
@@ -86,16 +90,16 @@ public boolean addKeyToSet(String key) {
     // reduce the number of keys by 1. if this number goes to zero then remove from the set
     // note: there is a good reason why we should not use a hashset but use hashmap instead
     //========================================================
-    public void removeKeyFromSet(String key) {
+    void removeKeyFromSet(String key) {
```

Comment:
if addKeyToSet is private, why is the opposite operation pacakge-private and not of the same visibility level?